addons/hr_expense/models/account_payment.py

```diff
@@ -33,7 +33,8 @@ def _synchronize_from_moves(self, changed_fields):
 
     def _synchronize_to_moves(self, changed_fields):
         # EXTENDS account
-        if self.expense_sheet_id:
+        trigger_fields = set(self._get_trigger_fields_to_synchronize()) | {'ref', 'expense_sheet_id', 'payment_method_line_id'}
+        if self.expense_sheet_id and any(field_name in trigger_fields for field_name in changed_fields):
```

Comment:
Wouldn't it be easier to simply extend `_get_trigger_fields_to_synchronize()` here in `hr_expense`?

Reply:
Not sure, as it's not the normal use of the _get_trigger_fields which is supposed to do the opposite
It feel safer to do it here imo, and whitelisting would stop studio from working